app/views/instrument_price_policies/_adjustment_row.html.haml

```diff
@@ -1 +1 @@
-= render "time_based_price_policies/adjustment_row", price_group: price_group, price_policy: price_policy, cancellation: true, minimum_cost: true
+= render "time_based_price_policies/adjustment_row", price_group: price_group, price_policy: price_policy, cancellation: true, minimum_cost: true, pp: pp
```

Comment:
It seems a bit strange to me to have these one-line partials which only render another partial. What do you think about updating the calls sites which render these partials, like [here](https://github.com/tablexi/nucore-open/pull/1707/files#diff-5ea7effd71575e1289a5e209029135beR32), to just do these render calls directly and eliminate the indirection?